dev-packages/application-package/src/application-props.ts

```diff
@@ -64,7 +64,8 @@ export namespace FrontendApplicationConfig {
         applicationName: 'Eclipse Theia',
         defaultTheme: 'dark',
         defaultIconTheme: 'none',
-        electron: ElectronFrontendApplicationConfig.DEFAULT
+        electron: ElectronFrontendApplicationConfig.DEFAULT,
+        defaultLocale: ''
```

Comment:
@msujew I'm a bit confused by this change, isn't something like `defaultLocale` specific to the end-user? 
I'm not sure I follow as to why application would set the `defaultLocale` for all their users as an application prop.

Reply:
As outlined in the community post, I can imagine it to be useful when deploying your app in a certain region, such as China, were you can assume that your user base is more likely to understand Chinese than English. Also, special installers might modify the `defaultLocale` (as in: Install this Theia app in English, German, French, etc.), were this change comes in handy. 

But otherwise I agree, it should be left to the choice of the end-user.